### PR TITLE
AdminCategory with edits

### PR DESCRIPTION
**Pull request by codegracie**
Please check my pull request.
Added changes and deleted files. I kinda messed up with git awhile back, so i had to create a new pull.

### CODE REVIEW

**Comment by kian1213** [comment]
Ok

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class Admin::CategoriesController < AdminController
+  def index
+    @categories = Category.all
+  end
+  def new
```
Comment (MuhammadTamzid) [inline app/controllers/admin/categories_controller.rb]: new line

File: app/controllers/admin/categories_controller.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    @categories = Category.new
+  end
+  def create
```
Comment (MuhammadTamzid) [inline app/controllers/admin/categories_controller.rb]: same above

File: app/controllers/admin/categories_controller.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  end
+  def edit
```
Comment (MuhammadTamzid) [inline app/controllers/admin/categories_controller.rb]: same above

File: app/helpers/admin/categories_helper.rb [hunk 4]
```diff
@@ -0,0 +1,2 @@
+module Admin::CategoriesHelper
```
Comment (dieunb) [inline app/helpers/admin/categories_helper.rb]: remove this file

File: app/controllers/admin/categories_controller.rb [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+  end
+  def show
```
Comment (MuhammadTamzid) [inline app/controllers/admin/categories_controller.rb]: same above

File: app/helpers/admin/categories_helper.rb (hunk 4 above)
Comment (MuhammadTamzid) [inline app/helpers/admin/categories_helper.rb]: remove unused file

File: app/views/admin/categories/edit.html.erb [hunk 6]
```diff
@@ -0,0 +1,16 @@
+<div id="page-wrapper" >
+  <div id="page-inner">
+    <hr/>  
+      <div class="row">
```
Comment (MuhammadTamzid) [inline app/views/admin/categories/edit.html.erb]: indent
remove two space

File: app/views/admin/categories/index.html.erb [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+                      <td>
+                        <div class="admin-actions">
+                          <%= link_to  admin_category_path(category), class: "button btn btn-success btn-xs" do %>
+                            <i class="fa fa-file-text"></i>
+                          <% end %>
+                          <%= link_to edit_admin_category_path(category), class: "button btn btn-primary btn-xs" do %>
+                            <i class="fa fa-pencil"></i>
+                          <% end %>
+                          <%= link_to admin_category_path(category), method: :delete,
+                          data: {confirm: "You want to delete?"},class: "button btn btn-danger btn-xs" do %>
```
Comment (MuhammadTamzid) [inline app/views/admin/categories/index.html.erb]: indent
add two space

File: app/controllers/admin/categories_controller.rb (hunk 1 above)
Comment (codegracie) [inline app/controllers/admin/categories_controller.rb]: noted po.

File: app/helpers/admin/categories_helper.rb (hunk 4 above)
Comment (codegracie) [inline app/helpers/admin/categories_helper.rb]: ok, deleted now.

**Pull request merged**